pkg/logs/auditor/auditor.go

```diff
@@ -34,35 +35,35 @@ type Auditor struct {
 	inputChan    chan message.Message
 	registry     map[string]*RegistryEntry
 	registryPath string
-
-	entryTTL time.Duration
-
-	done chan struct{}
+	mu           *sync.Mutex
```

Comment:
Why are you making that a pointer?

Reply:
You're right, it does not have to be a pointer.